modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts

```diff
@@ -167,6 +167,7 @@ const render = async (editor: Editor, uiRefs: ReadyUiReferences, rawUiConfig: Re
       if (Type.isNull(OuterContainer.whichView(outerContainer))) {
         editor.focus();
         editor.nodeChanged();
+        OuterContainer.refreshToolbar(outerContainer);
```

Comment:
Even if I remove this the test still passes. Does it need to be a manual test it is really hard to automatically test?

Reply:
ok, to make the test fail you need to comment this but also to revert the change in `OuterContainer` (put the `apis.refreshToolbar(comp);`), because the change in `OuterContainer` is needed to fix the case in the test:
```
TINY-9419: "Expand or collapse" button should not be removed if the toolbar is opened and view is opened and close
```
but this create the regression of this PR

Reply:
Right makes sense I can make the test failing now thanks.